db/migrations/2410292333_remove_unnecessary_enum_values_from_groups_propagate_ancestors_computation_state.sql

```diff
@@ -0,0 +1,5 @@
+-- +migrate Up
+ALTER TABLE `groups_propagate` MODIFY COLUMN `ancestors_computation_state` ENUM('todo', 'done') NOT NULL;
```

Comment:
As I already mentioned/asked in the past: Is it really better to have a todo/done and possibly not an entry for any existing group, versus no "state" column, just considering an entry to mean "todo". So basically deleting the rows instead of marking them "done"? It would allow the table to stay very small instead of converging to a size similar to the `groups` table. Maybe updating all is easier than deleting some and updating some ?

Reply:
Although it's very easy to implement now, I believe it would make the propagations slower because we would increase fragmentation of *_propagate table each time we delete rows from them. Currently we mostly update existing rows and never delete them preventing the fragmentation.